hedy.py

```diff
@@ -1605,7 +1609,7 @@ def transpile_inner(input_string, level):
                                                             invalid_argument=''.join(invalid_info.arguments))
                 # clearly the error message here should be better or it should be a different one!
                 raise ParseException(level=level, location=["?", "?"], keyword_found=invalid_command)
-            raise InvalidCommandException(invalid_command=invalid_command, level=level, guessed_command=closest)
+            raise hedy.InvalidCommandException(invalid_command=invalid_command, level=level, guessed_command=closest, location=[line, "?"])
```

Comment:
This is not necessary. This exception is already available in the current file, so you definitely don't need to namespace it.

I'm noticing other exceptions are doing it too... but honestly: we should remove the `import hedy` statement, and remove `hedy.` everywhere it's being used.

(But you can do that in a different PR as well, it doesn't really have bearing on the change you're making here)